### ISSUE TITLE
Feature request: Display the inaccessible alt text within the comment body

### ISSUE THREAD

**Issue by inkblotty** [github/accessibility-alt-text-bot#46]
Hi Accessibility-alt-text-bot team :)

### Context ❓ 

When the bot detects inaccessible alt text, or missing alt text, its target is generalized to "in the PR body" or "in the issue body", for example. However, if the issue body, for example, has multiple images, it's not clear which one is missing alt text from the comment itself.

### Idea 💡 

In the comment body, display which alt text is not accessible and/or which image has inaccessible / missing alt text.

**Comment by inkblotty** [comment]
Example inaccessible images for testing purposes.

First one:

![Image](https://github.com/github/accessibility-alt-text-bot/assets/14206003/c557a5dd-d4d3-4483-9419-1bc3d5e5ab61)

and a second one:

![Image](https://github.com/github/accessibility-alt-text-bot/assets/14206003/672dbe9e-45ff-4f3d-9fbc-6ca0ef5e1b9d)

**Comment by kendallgassner** [comment]
[Released in v1.6.0](https://github.com/github/accessibility-alt-text-bot/releases/tag/v1.6.0)
